nixos/modules/services/cluster/kubernetes/kubelet.nix

```diff
@@ -336,7 +336,10 @@ in
           [
             gitMinimal
             openssh
-            util-linux
+            # TODO (#409339): remove this patch. We had to add it to avoid a mass rebuild
```

Comment:
Not sure if I'm missing something, but I think the staging cycle for https://github.com/NixOS/nixpkgs/pull/405952 has to complete to switch back.

```suggestion
            # TODO (#405952): remove this patch. We had to add it to avoid a mass rebuild
```

Reply:
I just made that tracking issue to keep track of all the PRs where we changed something and some cleanup will be required (including the staging cycle): #409339

Reply:
Thx, makes sense

Reply:
I got it